src/app/dataset-view/additional-components/metadata-component/components/add-polling-source/steps/fetch-step/fetch-step.component.ts

```diff
@@ -30,12 +38,12 @@ export class FetchStepComponent extends BaseComponent implements OnInit {
 
     ngOnInit(): void {
         this.parentForm = this.rootFormGroupDirective.form;
-        this.initUrlFetchStep();
+        this.initForm("url");
```

Comment:
The defaults are repeating with the main component of the wizard, so move the defaults to constant data.

Reply:
Moved to constant data